discovery-infra/test_infra/consts/consts.py

```diff
@@ -189,6 +189,11 @@ class Events:
     PROXY_SETTINGS_CHANGED = "Proxy settings changed"
 
 
+class HostStatusInfo:
+    WRONG_BOOT_ORDER = "Expected the host to boot from disk, but it booted the installation image"
+    REBOOT_TIMEOUT = "Host failed to reboot within timeout"
```

Comment:
This is just part of the status info:
Full status info is:
```
"Host failed to reboot within timeout, please boot the host from the the OpenShift installation disk $INSTALLATION_DISK. The installation will resume once the host reboot"
```
Same for the wrong boot order, full status info is:
```
Expected the host to boot from disk, but it booted the installation image - please reboot and fix boot order to boot from disk $INSTALLATION_DISK
```
Consider renaming to REBOOT_TIMEOUT_PREFIX

Reply:
I dont have a way to pass the disk parameter, and anyway i dont think we need to validate the entire string, less maintenance in case wording is changes (same as we do with events). Note that there is also going to be a UI test for it. WDYT?